tools/add_triage_signature.py

```diff
@@ -2327,6 +2327,67 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("Controller logs contain some warnings:\n" + f"{{code}}{warning_text}{{code}}")
 
 
+class MissingOSTreePivot(ErrorSignature):
+    """
+    This signature looks for missing pivot URL in rpm-ostree status of the control-plane hosts
+    """
+
+    PIVOT_URL_PATTERN = re.compile(r"pivot://.*")
+    CONTROL_PLANE_PATH = "*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="missing_pivot_ostree",
+            comment_identifying_string="h1. MCO didn't pivot some hosts",
```

Comment:
do we want to negate on @javipolo signature?
https://github.com/openshift-assisted/assisted-installer-deployment/pull/328/files
although it's not so common, so not sure it worth the effort.